train-user/src/main/java/hu/bme/mit/train/user/TrainUserImpl.java

```diff
@@ -5,11 +5,26 @@
 
 public class TrainUserImpl implements TrainUser {
 
+	private static final long POLLING_INTERVAL = 200;
```

Comment:
Szép munka, de az intervall lehetne nagyobb is, ilyen gyakori frissitésre nincs szükség.